src/ensembl/src/shared/radio/Radio.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback } from 'react';
+
+import defaultStyles from './Radio.scss';
+
+export type radioOption = {
+  value: string;
+  label: string;
+};
+
+export type radioOptions = radioOption[];
+
+type Props = {
+  onChange: (selectedOption: string) => void;
+  classNames?: any;
+  radioOptions: radioOptions;
+  selectedOption: string;
+  disabled?: boolean;
+};
+
+const Radio = (props: Props) => {
+  const styles = props.classNames
+    ? { ...defaultStyles, ...props.classNames }
+    : defaultStyles;
+
+  const handleOnChange = useCallback(
+    (event: React.ChangeEvent<HTMLInputElement>) => {
+      if (props.disabled) {
+        return;
+      }
+      props.onChange(event.target.value);
+    },
+    [props.selectedOption]
+  );
+
+  return (
+    <>
```

Comment:
First, why fragment?
Second, why table? A radio button with a label should not be a table.

Reply:
Removed the fragment.

I preferred table here as I thought it will be easy to control the alignment of the radios. For example, at the moment it draws them horizontally, we can update this later on to support vertical alignment and multiple columns.